setup.cfg

```diff
@@ -0,0 +1,2 @@
+[mypy]
```

Comment:
You can put the implicit optional config in here with a comment like: https://github.com/app-sre/qontract-reconcile/blob/master/setup.cfg#L4

That will ensure that the option takes effect when people run `mypy` outside of `make` as well.

Reply:
Done.